WebRoot/css/zm.css

```diff
@@ -1483,6 +1487,22 @@ INPUT[type="text"].search_input-expanded:focus {
 	@AddreddBubbleLeftChild@
 }
 
+.attachmentBubble > :first-child {
+	@AttachmentLeftChild@
```

Comment:
In our code, have seen many places where we do styling based on child position and proceeded with  the same.